build

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
```

Comment:
Why not use a `Makefile` instead of a file to `build`, `install`, `clean`, `test` respectively?